internal/titles/bf1.go

```diff
@@ -35,8 +33,8 @@ var Bf1 = domain.GameTitle{
 		},
 	},
 	URLValidator: internal.MakePatternURLValidator(internal.Frostbite3GameIdPattern),
-	CmdBuilder:   internal.MakeOriginCmdBuilder("1026023"),
+	CmdBuilder:   internal.OriginCmdBuilder{},
 	HookHandlers: []game_launcher.HookHandler{
-		internal.MakeKillProcessHookHandler(false, bf1Exe), // Launcher config executable name will be "Origin.exe", which we don't want to kill
+		internal.MakeKillProcessHookHandler(true, bf1Exe),
```

Comment:
The second parameter here is obsolete now, since passing `true` for the first parameter (`targetLaunchExecutable`) already makes the hook target `bf1.exe`.